ssr/src/auth/mod.rs

```diff
@@ -44,15 +44,21 @@ impl DelegatedIdentityWire {
             signature: sig.signature.unwrap(),
         };
 
-        let mut delegation_chain = from.delegation_chain();
-        delegation_chain.push(signed_delegation);
-
         Self {
             from_key: sig.public_key.unwrap(),
             to_secret: to_secret.to_jwk(),
-            delegation_chain,
+            delegation_chain: vec![signed_delegation],
```

Comment:
Shouldn't this also include signed_delegation of `from_identity` (there could be a case where from is a delegated Identity)

Reply:
We create delegation for video importer from delegated Identity.